### PR TITLE
Audit translation tokens with a bash script

### PR DESCRIPTION
**Pull request by manzapanza**
Closes: #205

Run the script from root folder:
$ ./check-i18n-tokens.sh

For now, the script shows a list of i18n tokens that should be added on en.i18n.json file.

Next step:
- Show a list of i18n tokens that should be removed from en.i18n.json file

### CODE REVIEW

**Comment by brylie** [comment]
Very cool.

**Comment by manzapanza** [comment]
Now the script shows a list of i18n tokens that should be removed from en.i18n.json file.

Next step:

- Improve the script that generate the two lists considering also schema names fields

File: check-i18n-tokens.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+fileEnI18n="core/lib/i18n/en.i18n.json"
+
+echo "Tokens that should be added on en.i18n.json file:"
+echo ""
+
+# FIND TOKENS FROM HTML FILES
+tokensHtmlFiles=$(grep -iRoh '{{ *_ .*}}' * --exclude="*.md" | grep -o "['\"].*['\"]" | sed -e "s@['\"]@@g")
+for token in $tokensHtmlFiles; do
+  if ! grep -q "$token" "$fileEnI18n"; then
```
Comment (brylie) [inline check-i18n-tokens.sh]: Add comment describing these lines. We try to explain our code in plain English, just as we would to a novice programmer.

Comment (brylie) [inline check-i18n-tokens.sh]: I.e. simply describe what these two lines are trying to achieve.

File: check-i18n-tokens.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    echo "$token"
+  fi
+done
+
+# FIND TOKENS FROM JS FILES
+tokensJsFiles=$(grep -iRoh 'TAPi18n\.__(.*)' * --exclude="*.md" | grep -o "['\"].*['\"]" | sed -e "s@['\"]@@g")
+for token in $tokensJsFiles; do
+  if ! grep -q "$token" "$fileEnI18n"; then
```
Comment (brylie) [inline check-i18n-tokens.sh]: Add simple, descriptive comment here.
